libs/net/models/Areas/Services/Models/Content/ContentModel.cs

```diff
@@ -349,5 +349,15 @@ public ContentModel(Entities.Content entity, JsonSerializerOptions options) : ba
 
         return entity;
     }
+
+    public Entities.Content ToContent()
+    {
+        var content = (Entities.Content)this;
+        content.Source = this.Source != null ? (Entities.Source)this.Source : null;
```

Comment:
The default converter can't include these or they can result in bad updates in the database.